### ISSUE TITLE
get a11y plugin working

### ISSUE THREAD

**Issue by rosemm** [arcus/data_education_presentations#4]

**Comment by rosemm** [comment]
@franzenr So I actually worked on this for a WHILE and couldn't figure out how to make it work. It's a widely used plugin, but it's not one of the ones you can install directly (you have to download the pieces and put them in your plugin directory, which I cannot find...). I hit a wall with it. But if I can, I'll come back and troubleshoot again later, or you can feel free to take a stab if you like. One thing that's nice is that it's a theme, so we should be able to apply it later to already done presentations pretty easily. The theme I set in the template (night) is the one reveal-a11y is based on, so the appearance should be pretty similar.
